resources/artifact-ignores.properties

```diff
@@ -219,29 +219,29 @@ sonargraph-plugin = https://github.com/jenkins-infra/update-center2/pull/405
 sonatype-ci = https://wiki.jenkins.io/x/iYDPAw
```

Comment:
https://wiki.jenkins.io/display/JENKINS/63930505 perhaps?

Reply:
+1 from me to use that replacement.